google-cloud-spanner/lib/google/cloud/spanner/results.rb

```diff
@@ -85,6 +86,8 @@ def rows
 
           fields = @metadata.row_type.fields
           values = []
+          buffered_responses = []
+          buffer_upper_bound = 10
```

Comment:
Is `10` the best possible value for this bound? I have seen no suggested value in any documentation. I assume you got this from the google-cloud-node implementation since you mention it in the PR description. Is it just a matter of guessing for each platform?